### PR TITLE
Re-done-Alpha1,2 Beta1,2 tanycyte,NTR

### LINKED ISSUE #2256: [NTR] subtypes of tanycyte - alpha1,2; Beta 1,2
Please check that the term does not already exist by using the ontology search tool OLS:
https://www.ebi.ac.uk/ols4/ontologies/cl

**Preferred term label**

alpha 1 tanycyte
alpha 2 tanycyte
beta 1 tanycyte
beta 2 tanycyte

**Synonyms** (add reference(s), please)

**Definition** (free text, with reference(s), please. PubMed ID format is PMID:XXXXXX)

Review here: DOI: 10.1016/S0074-7696(05)47003-5

Table II has a good summary of differences in location, morphology and gene expression.  Looks like location and morphology might be sufficient to distinguish.

image

May be worth looking back at some more classical papers - review references EM studies from the 70s.

**Parent cell type term** (check the hierarchy here https://www.ebi.ac.uk/ols4/ontologies/cl)

**Anatomical structure where the cell type is found** (check Uberon for anatomical structures: https://www.ebi.ac.uk/ols4/ontologies/uberon)

**Your ORCID**

**Additional notes or concerns**

### PR DESCRIPTION
**Pull request by AvolaAmg**
fixes #2256

To follow changes in this PR, read @AmgAvola comments in the ticket #2256

Alpha1,2 Beta1,2 tanycyte added with definition, references, synonyms and logical definition.

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -32234,6 +32241,55 @@ SubClassOf(obo:CL_4042016 obo:CL_0000313)
...
+AnnotationAssertion(terms:contributor obo:CL_4042017 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(terms:date obo:CL_4042017 "2024-08-19T13:06:16Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:29351662") oboInOwl:hasRelatedSynonym obo:CL_4042017 "DMH tanycyte")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:29351662") oboInOwl:hasRelatedSynonym obo:CL_4042017 "VMH tanycyte")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:26578855") rdfs:comment obo:CL_4042017 "In rodents, a1 tanycytes are known to express DARPP-32, GPR50, NA Vimentin, Connexin 43, GLT-1, GLUT1(+) (Goodman and Hajihosseini 2015; Prevot et al., 2018).")
+AnnotationAssertion(rdfs:label obo:CL_4042017 "alpha1-tanycyte"@en)
+SubClassOf(obo:CL_4042017 obo:CL_0002085)
+SubClassOf(obo:CL_4042017 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0004642))
```
[the remark this answers is not in the document]
Comment (AvolaAmg) [reply src/ontology/cl-edit.owl:32254]: has_soma_location should be only to neurons right? 
this refers to 'part_of' and in the case of alpha1 tanycyte the anatomical part of the 'third ventricle ependyma'
alpha2 is in the 'arcuate nucleus of hypothalamus' 
beta1 and beta2 are in the 'infundibular recess of 3rd ventricle'
Do you mean I should have all of the in the 'third ventricle ependyma? 
thank you!

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -32234,6 +32241,55 @@ SubClassOf(obo:CL_4042016 obo:CL_0000313)
...
+AnnotationAssertion(rdfs:label obo:CL_4042018 "alpha2-tanycyte"@en)
+SubClassOf(obo:CL_4042018 obo:CL_0002085)
+SubClassOf(obo:CL_4042018 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001932))
+SubClassOf(obo:CL_4042018 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000014419))
+
+# Class: obo:CL_4042019 (beta1-tanycyte)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:16344112") Annotation(oboInOwl:hasDbXref "PMID:26578855") Annotation(oboInOwl:hasDbXref "PMID:29351662") Annotation(oboInOwl:hasDbXref "Wikipedia:Tanycyte") obo:IAO_0000115 obo:CL_4042019 "A type of tanycyte that has its soma located in the lateral infundibular recess of the brain. This tanycyte has an elongated morphology with multiple microvilli extending to the third ventricle in the lateral zones of the median eminence. This type of tanycyte expresses FGF receptors 1 and 2, is in contact with GnRH neurons, and is involved in the release of gonadotropin-releasing hormone (GnRH).")
```
[the remark this answers is not in the document]
Comment (AvolaAmg) [reply src/ontology/cl-edit.owl:32271]: they express FGFR1 in all of the beta1 tanycytes and beta 2 tanycytes (from literature evidence)

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -3109,9 +3113,12 @@ Declaration(Class(obo:PR_000004919))
 Declaration(Class(obo:PR_000005444))
 Declaration(Class(obo:PR_000005460))
 Declaration(Class(obo:PR_000007204))
+Declaration(Class(obo:PR_000007479))
```
[the remark this answers is not in the document]
Comment (AvolaAmg) [reply src/ontology/cl-edit.owl:3116]: thank you!

File: src/ontology/cl-edit.owl (hunk 1 above)
[the remark this answers is not in the document]
Comment (aleixpuigb) [reply src/ontology/cl-edit.owl:32254]: That is correct, it should be used in neurons. But your textual definition uses this terminology:
"A tanycyte cell that has its soma located in the ventromedial and dorsomedial nuclei of the hypothalamus."
Probably the definition should be amended.

[the remark this answers is not in the document]
Comment (AvolaAmg) [reply src/ontology/cl-edit.owl:32254]: my mistake I will correct this, did not even notice I did that thank you!

**Review by aleixpuigb — APPROVED** [review]

**Review by aleixpuigb — CHANGES_REQUESTED** [review]
The branch has conflicts.

**Comment by dosumis** [comment]
#gogoeditdiff (not sure if this will work once closed, but trying as will be useful for me grok details of changes)

**Pull request merged**